docs/tutorial_easybuild.rst

```diff
@@ -1,5 +1,5 @@
 ============================================
-Tutorial 2: Using the EasyBuild build system
+Tutorial 5: Using the EasyBuild build system
```

Comment:
```suggestion
Tutorial 5: Using package managers as a build system
```

Reply:
And generally I would remove the mention of EasyBuild in this introductory section, because we are going to add another section about Spack.